src/did/resolvers/example.rs

```diff
@@ -1,59 +1,28 @@
 use async_trait::async_trait;
 
 use crate::{
-    did::{DIDDoc, DIDResolver, Service, VerificationMethod},
+    did::{DIDDoc, DIDResolver},
     error::Result,
 };
 
 /// Allows resolve pre-defined did's for `example` and other methods.
 pub struct ExampleDIDResolver {
-    known_dids: Vec<ExampleDIDDoc>,
+    known_dids: Vec<DIDDoc>,
```

Comment:
I think it will better to rename `ExampleDIDResolver` to `DIDDocResolverInMemory`. Anyway as I understand this crate should resolve `DID Doc` not `DID`

Reply:
I disagree. You resolves `did` to `diddoc`. See specification

`
The DID resolution functions resolve a DID into a DID document by using the "Read" operation of the applicable DID method as described in § 8.2 Method Operations. The details of how this process is accomplished are outside the scope of this specification, but all conforming DID resolvers implement the functions below, which have the following abstract forms:
`

Reply:
Let's keep it as is and discuss with Alexa and team next week